src-docs/src/views/range/range_example.js

```diff
@@ -238,6 +238,14 @@ export const RangeControlExample = {
             <EuiCode>label</EuiCode>. The value must be included in the range of
             values (min-max), though the label may be anything you choose.
           </p>
+          <p>
+            The <EuiCode>EuiRangeTick</EuiCode> interface now includes an
+            optional <EuiCode>accessibleLabel</EuiCode>. This property is
+            combined with the current <EuiCode>value</EuiCode> to render an{' '}
+            <EuiCode>aria-valuetext</EuiCode> attribute. A{' '}
+            <EuiCode>label</EuiCode> of type string will be combined with the
+            current value when no accessible label is passed.
+          </p>
```

Comment:
I found this copy hard to read/understand and a little too technical (describing the "how" of this PR rather than "why" it matters or "what" to do). Let me know whether you think the below copy is an improvement?
```suggestion
          <p>
            Tick labels can improve the accessibility of your range. If your
            label is a simple string, it will be read out to screen readers
            alongside the value. You can also use the{' '}
            <EuiCode>accessibleLabel</EuiCode> property to provide more explicit
            screen reader text.
          </p>
```

Reply:
Also this is my last comment I swear, this PR is good to merge after! :)